docs/evaluation.md

```diff
@@ -10,100 +10,169 @@ data including inputs, outputs, and information from intermediate steps to
 evaluate the end-to-end quality of LLM responses as well as understand the
 performance of your system's building blocks.
 
-For example, if you have a RAG flow, Genkit will extract the set of documents
-that was returned by the retriever so that you can evaluate the quality of your
-retriever while it runs in the context of the flow as shown below with the
-Genkit faithfulness and answer relevancy metrics:
+### Types of evaluations
 
-```ts
-import { genkit } from 'genkit';
-import { genkitEval, GenkitMetric } from '@genkit-ai/evaluator';
-import { vertexAI, textEmbedding004, gemini15Flash } from '@genkit-ai/vertexai';
+Genkit supports two types of evaluations:
 
-const ai = genkit({
-  plugins: [
-    vertexAI(),
-    genkitEval({
-      judge: gemini15Flash,
-      metrics: [GenkitMetric.FAITHFULNESS, GenkitMetric.ANSWER_RELEVANCY],
-      embedder: textEmbedding004, // GenkitMetric.ANSWER_RELEVANCY requires an embedder
-    }),
-  ],
-  // ...
-});
-```
+1. **Inference-based evaluation**: In this type of evaluation, a system is run on a collection of pre-determined inputs and the corresponding outputs are assessed for quality. 
```

Comment:
"A system is run on" sounds strange to me. Maybe we just say, "This type of evaluation is run against a collection of..." or similar

Reply:
s/1./* 

Numbered lists are really more for sequences (i.e., procedures, etc.).